errormessage/src/main/kotlin/motif/errormessage/CannotResolveTypeHandler.kt

```diff
@@ -0,0 +1,17 @@
+package motif.errormessage
+
+import motif.models.CannotResolveType
+
+class CannotResolveTypeHandler(private val error: CannotResolveType) : ErrorHandler {
```

Comment:
So far, I only know that we need to add license header like [this new added file](https://github.com/uber/motif/pull/227/files#diff-e95cdc9b4b6771bd0aa35517e4d54da58d25a9321014068b82e4ada9157209fc) to make its corresponding unit test pass.

Reply:
Thanks, it seems like running the tests locally, auto-generates and resolves that issue :)